LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 Evans kofi Nyamekye
```

Comment:
- [ ] Kindly make sure that you have added the `relevant` information in the `LICENSE` file because it seems like you have added the `LICENSE` on behalf of someone in your application, which means you are giving the credit to your application and work "Evans Kofi Nyamekye".